src/api/v1/db/oracledb/setupDb.sql

```diff
@@ -0,0 +1,72 @@
+DROP TABLE PIZZA_INGREDIENTS;
+DROP TABLE PIZZAS;
+DROP TABLE INGREDIENTS;
+DROP TABLE DOUGHS;
+
+CREATE TABLE "DOUGHS" 
+   (	"ID" NUMBER GENERATED ALWAYS AS IDENTITY MINVALUE 1 MAXVALUE 9999999999999999999999999999 INCREMENT BY 1 START WITH 1 CACHE 20 NOORDER  NOCYCLE  NOT NULL, 
+	"NAME" VARCHAR2(256 BYTE) NOT NULL, 
+	"GRAMS_FLOUR" NUMBER(*,0) NOT NULL, 
```

Comment:
this is generated by sql developer to indicate an integer. Arbitrary precision seems fine to me but there is a relatively low theoretical cap for all of these value so we could specify a smaller precision if we wanted to go there.

Reply:
According to [this](https://www.oracletutorial.com/oracle-basics/oracle-number-data-type/) it has the maximum range and precision by default